app/views/admin_v2/team/_member.html.erb

```diff
@@ -19,20 +19,29 @@
       <% end %>
 
       <% dropdown.with_item do %>
```

Comment:
Yep, I'm aware of this one. I wanted an action to be available on all member statuses until all the actions were added. It's been a big help while building out the UI. But it can be put in its correct place now.

Just an FYI, pending member drop downs will display this for now:
<img width="325" alt="Screenshot 2024-07-10 at 08 46 06" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/b8d6e7af-2122-417a-af2a-c6935da39dbe">


It's not a bug! lol. We have a [couple of actions](https://github.com/orgs/TheOdinProject/projects/46) still left to do for pending members.